doc/nrf/releases/release-notes-latest.rst

```diff
@@ -203,6 +203,23 @@ The following list summarizes the most important changes inherited from upstream
   * Removed all nRF-specific aliases to particular hardware peripherals, because they are no longer needed now that nodes can be addressed by node labels.
     For example, you should now use ``DT_NODELABEL(i2c0)`` instead of ``DT_ALIAS(i2c_0)``.
 
+* Mcuboot:
+
+  * Fix erase of possible trailer leftover after reset
+  * Added option to relocate interrupts to application
+  * Fix single image compilation with serial recovery
+  * Allow single image applications
+  * Allow disable cryptographic check of the image
+  * serial_recovery: Reduce minimum number of members in packet
+  * Introduced direct execute-in-place (XIP) mode
+  * Fix kramdown CVE-2020-14001
+  * Allow to point zephyr project related key via Kconfig (Use private key)
+  * boot: Support for nrf52840 with ecc keys and cryptocell
+  * Allow to set VTOR to relay vector before chain-load the application
+  * Allow larger primary slot in swap-move
```

Comment:
Any references?

Reply:
This was not patch to the move algo, but to before check. Added more description on that.
https://github.com/JuulLabs-OSS/mcuboot/pull/760